<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Dell-C193V93" failures="0" tests="2" name="mohamedNProject.tests.SubmitOrderTest" time="0.000" errors="1" timestamp="2024-11-09T13:58:22 IST" skipped="1">
  <testcase classname="mohamedNProject.tests.SubmitOrderTest" name="OrderHistoryTest" time="0.000">
    <skipped/>
  </testcase> <!-- OrderHistoryTest -->
  <testcase classname="mohamedNProject.tests.SubmitOrderTest" name="submitOrder" time="0.000">
    <error message="java.nio.file.NoSuchFileException: D:\Software Tester\Automation Testing\MohdMN\SeleniumFrameworkDesign\src\test\java\rahulshettyacademy\data\PurchaseOrder.json" type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException: java.nio.file.NoSuchFileException: D:\Software Tester\Automation Testing\MohdMN\SeleniumFrameworkDesign\src\test\java\rahulshettyacademy\data\PurchaseOrder.json
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:58)
at org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:159)
at org.testng.internal.Parameters.handleParameters(Parameters.java:809)
at org.testng.internal.Parameters.handleParameters(Parameters.java:734)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:946)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.nio.file.NoSuchFileException: D:\Software Tester\Automation Testing\MohdMN\SeleniumFrameworkDesign\src\test\java\rahulshettyacademy\data\PurchaseOrder.json
at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:236)
at java.base/java.nio.file.Files.newByteChannel(Files.java:380)
at java.base/java.nio.file.Files.newByteChannel(Files.java:432)
at java.base/java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:422)
at java.base/java.nio.file.Files.newInputStream(Files.java:160)
at org.apache.commons.io.FileUtils.lambda$readFileToString$12(FileUtils.java:2686)
at org.apache.commons.io.IOUtils.toString(IOUtils.java:3221)
at org.apache.commons.io.IOUtils.toString(IOUtils.java:3196)
at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:2686)
at mohamedNProject.TestComponents.BaseTest.getJsonDataToMap(BaseTest.java:79)
at mohamedNProject.tests.SubmitOrderTest.getData(SubmitOrderTest.java:70)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:54)
... 25 more
]]>
    </error>
  </testcase> <!-- submitOrder -->
</testsuite> <!-- mohamedNProject.tests.SubmitOrderTest -->
